### PR TITLE
Feature/551 bri 3 messaging component model and create

### PR DESCRIPTION
**Pull request by ognjenkurtic**
# Description

Introducing db schema, migration, domain object, commands and command handlers for BpiMessages. COntroller with CRUD endpoints and tests are following in subsequent PRs. 

## Related Issue

#551 

## Motivation and Context

Basis for BpiMessages CRUD and later Bpi Communication model

## How Has This Been Tested

The app compiles and runs. Unit tests and postman collection will follow. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

File: examples/bri-3/prisma/schema.prisma [hunk 1]
```diff
@@ -40,3 +41,14 @@ model Transaction {
   signature String
   status Int
 }
+
+model Message {
+  id String @id
+  fromBpiSubjectId String
+  fromBpiSubject BpiSubject @relation(fields: [fromBpiSubjectId], references: [id], name: "fromBpiSubject_fk")
```
Comment (skosito) [inline examples/bri-3/prisma/schema.prisma:48]: i think convention is capital first letter according to their docs

File: examples/bri-3/src/bri/communication/capabilities/createBpiMessage/createBpiMessageCommand.handler.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  ) {}
+
+  async execute(command: CreateBpiMessageCommand) {
+    const { fromBpiSubject, toBpiSubject } =
+      await this.agent.getFromAndToSubjectsAndThrowIfNotExist(
+        command.fromId,
+        command.toId,
+      );
+
+    const newBpiSubjectCandidate = this.agent.createNewBpiMessage(
```
Comment (skosito) [inline examples/bri-3/src/bri/communication/capabilities/createBpiMessage/createBpiMessageCommand.handler.ts:22]: bpiSubject in name, replace with message
not important but maybe createNewBpiMessage and other create methods in non-storage agent should also be named something like createXCandidate

**Review by skosito — APPROVED** [review]

File: examples/bri-3/src/bri/communication/capabilities/createBpiMessage/createBpiMessageCommand.handler.ts (hunk 2 above)
Comment (ognjenkurtic) [reply examples/bri-3/src/bri/communication/capabilities/createBpiMessage/createBpiMessageCommand.handler.ts]: Fixed var name. I ll do the rename of methods in a separate small PR

File: examples/bri-3/prisma/schema.prisma (hunk 1 above)
Comment (ognjenkurtic) [reply examples/bri-3/prisma/schema.prisma]: Fixed

**Review by skosito — APPROVED** [review]

**Pull request merged**